bogota-july-2018/Calculator-JS/DaniloPlazas/Calculator HTML/scriptJS.js

```diff
@@ -0,0 +1,129 @@
+var num_press = 0.0; 	//Number pressed by the user
+var opt_press = ""; 	//Arithmetic Operation pressed by the user
+var num_acum = 0.0;		//Accumulated Number
+var opt_pend = ""; 		//Arithmetic Operation pending execution
+
+/*
```

Comment:
Nice documentation but the default format for this is the following:
```
/**
 * Description of the method and posible values:
 *
 * true - some value
 * false - other meaning
 * object -because we need an object for something
 *
 * @param defaultValue
 * @param params
 * @param someOther
 * @returns {boolean|Object}
 */
```

Reply:
Keep in mind this is optional, but just because u wanted to doc your functions. Good one anyway